### PR TITLE
Loading the individuals index (in elasticsearch) via a periodic celery task

### PR DESCRIPTION
**Pull request by mmulich**
## Pull Request Overview

This replaces the logstash ingestion of data from wildbook with a python based alternative. This change contains a celery task to be run every 10mins that will query marked-individuals in the wildbook database and put them in the individuals index.

This is a 1:1 feature replacement. Additionally it gives us space to bridge that data with data we have in houston (e.g. encounters and users information). That is not currently implemented in this PR, but you can see where that would happen in a comment within the `load_indexes` function.

Moving this logic into python is likely a de-optimization of the overall process. However, the benefits of bridging the data outweigh the difficulties of bridging this data gap in logstash. We'll be able to optimize the process at a later time.

### CODE REVIEW

**Review by colinwkingen — APPROVED** [review]
LGTM. Will this save us anything significant in resources? I don't remember logstash consuming too much memory but not sure.

File: app/modules/elasticsearch/tasks.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            indv = Individual(**result)
+            # Assign the elasticsearch document identify
+            indv.meta.id = f'markedindividual_{indv.id}'
+            # Augment records with houston data points
+            pass
+            # Save document to elasticsearch
+            indv.save(using=current_app.elasticsearch)
+
+
+WILDBOOK_MARKEDINDIVIDUAL_SQL_QUERY = """\
```
Comment (naknomum) [inline app/modules/elasticsearch/tasks.py:43]: this is one heck of a sql statement.  kudos.  i feel like i followed about 90% of it, and learned from the other 10% !

**Pull request merged**